lib/uffizzi/cli/login.rb

```diff
@@ -41,11 +40,10 @@ def handle_failed_response(response)
     end
 
     def handle_succeed_response(response)
-      unless account_valid?(response[:body][:user][:accounts].first)
-        puts 'No account related to this email'
-        return
-      end
-      account_id = response[:body][:user][:accounts].first[:id]
+      account = response[:body][:user][:accounts].first
+      return puts 'No account related to this email' unless account_valid?(account)
+
+      account_id = account[:id]
```

Comment:
This method is very simple. So initializing an additional variable here is redundant

Reply:
Ah, I see!